website/content/docs/discovery/services.mdx

```diff
@@ -275,6 +275,255 @@ For Consul 0.9.3 and earlier you need to use `enableTagOverride`. Consul 1.0
 supports both `enable_tag_override` and `enableTagOverride` but the latter is
 deprecated and has been removed as of Consul 1.1.
 
+### Tagged Addresses
+
+Tagged addresses are additional addresses which may be defined for a node or
+service. Tagged addresses can be used by remote agents and services as alternative
+addresses for communicating with the given node or service. Multiple tagged
+addresses address may be configured on a node or service.
+
+The syntax for defining a tagged address is as follows.
+
+<CodeTabs heading="Tagged address format">
+
+```hcl
+service {
+  name = "redis"
+  port = 80
+  tagged_addresses {
+    <tag> = {
+      address = "<address>"
+      port = port
+    }
+  }
+}
+```
+
+```json
+{
+  "service": {
+    "name": "redis",
+    "port": 80,
+    "tagged_addresses": {
+      "<tag>": {
+        "address": "<address>",
+        "port": port
+      }
+    }
+  }
+}
+```
+
+</CodeTabs>
+
+Consul supports several tagged address types. The various types and their use
+cases are detailed below.
+
+#### LAN
+
+LAN addresses are intended to be directly accessible only from services within
+the same Consul data center.
+
+##### Supported tags
+
+* `lan` - The IPv4 LAN address at which the node or service is accessible.
+* `lan_ipv4` - The IPv4 LAN address at which the node or service is accessible.
+* `lan_ipv6` - The IPv6 LAN address at which the node or service is accessible.
+
+<CodeTabs heading="Example LAN tagged address configuration">
+
+<CodeBlockConfig filename="redis-service.hcl">
+
+```hcl
+service {
+  name = "redis"
+  address = "192.0.2.10"
+  port = 80
+  tagged_addresses {
+    lan = {
+      address = "192.0.2.10"
+      port = 80
+    }
+    lan_ipv4 = {
+      address = "192.0.2.10"
+      port = 80
+    }
+    lan_ipv6 = {
+      address = "2001:db8:1:2:cafe::1337"
+      port = 80
+    }
+  }
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig filename="redis-service.json">
+
+```json
+{
+  "service": {
+    "name": "redis",
+    "address": "192.0.2.10",
+    "port": 80,
+    "tagged_addresses": {
+      "lan": {
+        "address": "192.0.2.10",
+        "port": 80
+      },
+      "lan_ipv4": {
+        "address": "192.0.2.10",
+        "port": 80
+      },
+      "lan_ipv6": {
+        "address": "2001:db8:1:2:cafe::1337",
+        "port": 80
+      }
+    }
+  }
+}
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+#### Virtual
+
+Virtual tagged addresses are a logical address type which is used by
+[Connect](/docs/connect) to route traffic to the associated destination service
+in the service mesh when:
```

Comment:
Just to be clear, both conditions should be met to route traffic--not either or, correct? If so, consider making it more explicit.

Reply:
Yes, both conditions are required to be met. I can add such phrasing to the previous sentence.